src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -62,80 +83,47 @@
 import java.util.logging.Level;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.io.output.TeeOutputStream;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.*;
-
-import org.apache.commons.lang.StringUtils;
-import org.jenkinsci.test.acceptance.docker.Docker;
-import org.jenkinsci.test.acceptance.docker.DockerContainer;
-import org.jenkinsci.test.acceptance.docker.DockerRule;
-import org.jenkinsci.test.acceptance.docker.fixtures.JavaContainer;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-import static org.junit.Assume.*;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
-import org.jvnet.hudson.test.Issue;
-import org.jvnet.hudson.test.JenkinsRule;
-import org.jvnet.hudson.test.LoggerRule;
-import org.jvnet.hudson.test.SimpleCommandLauncher;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.emptyString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 enum TestPlatform {
     ON_CONTROLLER, NATIVE, ALPINE, CENTOS, UBUNTU, NO_INIT, UBUNTU_NO_BINARY, SLIM
 }
 
-@RunWith(Parameterized.class)
-public class BourneShellScriptTest {
-    @Parameters(name = "{index}: {0}")
-    public static Object[] data() {
-        return TestPlatform.values();
-    }
+@WithJenkins
+@ParameterizedClass(name = "{index}: {0}")
```

Comment:
Not sure what exactly is responsible, but this regresses proper reporting of test results in Jenkins. Compare https://ci.jenkins.io/job/Plugins/job/durable-task-plugin/job/master/lastSuccessfulBuild/testReport/org.jenkinsci.plugins.durabletask/BourneShellScriptTest/ to https://ci.jenkins.io/job/Plugins/job/durable-task-plugin/job/PR-274/3/testReport/org.jenkinsci.plugins.durabletask/BourneShellScriptTest/ where the latter fails to note which platform each test result belongs to.

TBH I am pretty sick of parameterization frameworks; if I were writing this test now, I would just use plain old Java code (on JUnit 4), maybe with `ErrorCollector` but maybe just with one `@Test` per meaningful combination.